src/components/ha-sidebar.ts

```diff
@@ -35,6 +41,7 @@ import {
   ExternalConfig,
   getExternalConfig,
 } from "../external_app/external_config";
+import { actionHandler } from "../panels/lovelace/common/directives/action-handler-directive";
```

Comment:
Can you check what the impact is of this PR on the sidebar chunk size?

Reply:
Just realized that we do load the sidebar async in `home-assistant-main`, so it won't impact our initial bundle, just the visibility of the sidebar.

Reply:
Before:
![image](https://user-images.githubusercontent.com/5662298/91856912-76f86b80-ec67-11ea-88d7-ed2b74230a1b.png)
After:
![image](https://user-images.githubusercontent.com/5662298/91856608-1406d480-ec67-11ea-8275-cba2eb40cae1.png)